src/interfaces/common.ts

```diff
@@ -48,6 +48,7 @@ export enum AnalyticsTrackEventsEnum {
 }
 
 export enum AnalyticsTrackSource {
+	Default = "default",
```

Comment:
why we need `default` tracks source?

Reply:
Well, from the splunk log it seems we always need a source for TrackEvent. 
Some of this event can be for cloud or ghe, not sure what to put. So `Default` comes to my mind.

Reply:
how about `gitHubCloud` instead of `default`?

Reply:
Good point, I've updated to use cloud now.